src/components/Context/injectWeb.js

```diff
@@ -19,7 +19,7 @@ export default window => {
         URL: window.location.href || window.location
       },
       webReferrer: {
-        URL: window.top.document.referrer
+        URL: window.document.referrer
```

Comment:
Should I create a doc ticket to mention this change?

Reply:
All the current docs say is this: "The URL of the previous page visited." so we probably don't need to rush out a doc change. I think the release notes will be fine. It would be good to have a ticket to to mention how an iframe will affect this value in the docs.